swan-lake/resources/featured-scenarios/create-your-first-connector-with-ballerina.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: ballerina-create-your-first-connector-left-nav-pages-swanlake
+title: Create your first connector with Ballerina
+description: step-by-step guide to create Ballerina connectors using OpenAPI specifications.
+keywords: ballerina, connector, library, REST, API, openAPI, Ballerina Central
+permalink: /learn/create-your-first-connector-with-ballerina/
+active: create-your-first-connector-with-ballerina
+intro: This guide walks you through creating your first Ballerina connector using an OpenAPI specification.
+---
+
+## Introduction
+
+Ballerina is a programming language that simplifies integration by providing a large library of pre-built connectors. These connectors are special packages consisting of one or more [Ballerina clients](https://ballerina.io/learn/by-example/client-class/), which allow communication with external services, usually via REST APIs. By using connectors, developers can quickly integrate third-party services into their Ballerina applications without having to worry about the technical details of API interactions.
+
+Along with its powerful library ecosystem, Ballerina also allows developers to easily create, share, and manage client connectors. These connectors are typically published on Ballerina Central as public connectors, available to the entire community, but you can also publish them as private connectors for internal use within your organization.
+
+In this guide, we'll walk you through how to generate your first Ballerina connector using an OpenAPI specification. This is one of the fastest and easiest ways to build connectors, enabling you to quickly integrate external services into your Ballerina projects.
+
+## Set up the prerequisites
```

Comment:
IMO `Prerequisites` would be enough as it is concise and clear.

Reply:
+1. updated in https://github.com/ballerina-platform/ballerina-dev-website/pull/9434/commits/6dc1f93b6ef0158c9cec6637e341db136b2c5385